docs/multiversion-multiinstance-smoke-test.md

```diff
@@ -0,0 +1,53 @@
+I am not sure a PR is necessary:
+
+## Setup
+
+1. Install ServiceControl version 2.
+1. Add 1 instance of ServiceControl v2 with MSMQ. It would help to use a name relating it to v3, e.g. `v2_SC`
+   - Configure a unique error queue name. It would help to use a name relating it to v2, e.g. `v2_error`
+   - Configure a unique audit queue name. It would help to use a name relating it to v2, e.g. `v2_audit`
+1. Install ServiceControl version 3
+1. Add 1 instance of ServiceControl v3 with MSMQ. It would help to use a name relating it to v3, e.g. `v3_SC`
+   - Configure a unique error queue name. It would help to use a name relating it to v3, e.g. `v3_error`
+   - Configure a unique audit queue name. It would help to use a name relating it to v3, e.g. `v3_audit`
+1. Download or checkout the [FaultTolerance](https://docs.particular.net/samples/faulttolerance/) sample.
+1. Edit the configuration of the sample project
+   - Reconfigure the endpoint to use the MSMQ transport
+   
+## Scenario 1   
```

Comment:
Should we label these scenarios somehow? i.e. "V2 remote notifies V3 master about successful retry"